File: sem4/MPP/mpp-proiect-repository-mariaruncan/ProjJavaFX/Model/src/main/java/app/model/dto/DtoUtils.java
package app.model.dto;

import app.model.Employee;

public class DtoUtils {
    public static Employee getFromDTO(EmployeeDto empdto){
        String username = empdto.getUsername();
        String password = empdto.getPassword();
        return new Employee(username, password);
    }
    public static EmployeeDto getDTO(Employee emp){
        String username = emp.getUsername();
        String pass = emp.getHashedPassword();
        return new EmployeeDto(username, pass);
    }
}

File: sem4/MPP/mpp-proiect-repository-mariaruncan/ProjCSharp/Model/dto/DtoUtils.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Model.dto
{
    public class DtoUtils
    {
        public static Employee GetFromDTO(EmployeeDto empDto)
        {
            string username = empDto.Username;
            string password = empDto.Password;
            return new Employee(username, password);
        }

        public static EmployeeDto getDTO(Employee emp)
        {
            string username = emp.Username;
            string pass = emp.HashedPassword;
            return new EmployeeDto(username, pass);
        }
    }
}
